package.json

```diff
@@ -14,12 +14,14 @@
   "dependencies": {
     "discord.js": "^11.5.1",
     "dotenv": "^8.1.0",
-    "firebase": "^7.1.0",
+    "firebase": "^7.2.0",
     "handlebars": "^4.4.3",
     "module-alias": "^2.2.2",
     "node-fetch": "2.6.0",
     "request": "^2.88.0",
-    "request-promise-native": "^1.0.7"
+    "request-promise-native": "^1.0.7",
+    "zlib-sync": "^0.1.5",
```

Comment:
What is `zlib-sync` and `zucc` used for because you aren't using them?

Reply:
I've seen them after npm install, discordjs recommend these optionals libs but in fact i don't need them the code work already fine. i will remove these Deps

Reply:
Thanks